app/src/main/java/com/mobileproto/dabrahamsmruehle/scavengerhunt/AWS_Video.java

```diff
@@ -0,0 +1,138 @@
+package com.mobileproto.dabrahamsmruehle.scavengerhunt;
+
+import android.app.Activity;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.MediaController;
+import android.widget.Toast;
+import android.widget.VideoView;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link AWS_Video.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link AWS_Video#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class AWS_Video extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
```

Comment:
Just get rid of these for now, or comment them. You shouldn't have unneccessary code merged into master